handlers.go

```diff
@@ -153,9 +152,10 @@ func (s *Server) HandleWebsocket(w http.ResponseWriter, r *http.Request) {
 
 				switch typ {
 				case "EVENT":
+					latestInex := len(request) - 1
 					// it's a new event
 					var evt nostr.Event
-					if err := json.Unmarshal(request[1], &evt); err != nil {
+					if err := json.Unmarshal(request[latestInex], &evt); err != nil {
```

Comment:
What do you intend on this?

Reply:
Hey @mattn, I  am using the last index because the JSON object is not always available in the index 1. If you check the sepc [01](https://github.com/nostr-protocol/nips/blob/master/01.md#from-relay-to-client-sending-events-and-notices), position 1 in index is the subscription ID (if exists). Which makes the JSON object comply with the pattern, being the latest item in the array

Reply:
Ah, right.

Reply:
This is wrong. If the the REQ doesn't have a subscription id it is invalid and should be ignored.

`go-nostr` has a new unmarshaler for these requests that should take care of these issues and present these messages in a friendlier way with proper structs.

Reply:
Hey @fiatjaf, it is not the REQ but the EVENT type

Reply:
Ah, sorry, that makes sense them. So this was a horrible bug?

Reply:
I took me a while to realize the reason why my transactions were not accepted by the relayer was the indexing of the objects. But I am happy to provide the solution so people don't fall on it again.